Integrations/integration-ServiceNow.yml

```diff
@@ -828,7 +828,8 @@ script:
     - name: template
       description: Template name to use as a base to create new incidents.
     - name: custom_fields
-      description: 'Custom fields in this format: fieldname=value;fieldname=value...'
+      description: 'Custom fields in this format: fieldname=value;fieldname=value...
+        (You don''t need to add "u_" in the name of the custom field. It adds it automatically.'
```

Comment:
shouldn't `don''t`  be `don\'t`?

Reply:
I wrote 
You don't need to add "u_" in the name of the custom field. It adds it automatically.
Demisto added the duplicate ''